src/jaegertracing/DynamicLoad.cpp

```diff
@@ -0,0 +1,26 @@
+#include <cstring>
+#include <system_error>
+
+#include <opentracing/dynamic_load.h>
+
+#include "TracerFactory.h"
+#include "jaegertracing/Tracer.h"
+
+int OpenTracingMakeTracerFactory(const char* opentracingVersion,
+                                 const void** errorCategory,
+                                 void** tracerFactory)
```

Comment:
Can you add `assert(tracerFactory != nullptr)`?

Reply:
Added.